inc/saml/namespace.php

```diff
@@ -45,6 +45,12 @@ function get_idp_metadata_file_path() : string {
 		return Altis\ROOT_DIR . DIRECTORY_SEPARATOR . $config['metadata_file'];
 	}
 
+	// Check for env-specific metadata files.
+	$env_file = Altis\ROOT_DIR . '/.config/sso/' . sprintf( 'saml-idp-metadata-%s.xml', \Altis\get_environment_type() );
```

Comment:
```suggestion
	$env_file = Altis\ROOT_DIR . '/.config/sso/' . sprintf( 'saml-idp-metadata-%s.xml', Altis\get_environment_type() );
```

Reply:
Are you ok with this not being backported as it's a new addition to functionality? Approving but will leave it to you to merge depending on your decision.

Reply:
@roborourke I think the best way forward here is to create another documentation PR to cover earlier versions, with a slightly modified text. Should I do that first, get it merged and backported, then we can merge this one here without backporting it ?